bundles/best_practices.rst

```diff
@@ -177,16 +166,23 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Installation
+------------
+
+Bundles must define a `Symfony Flex recipe`_ to automate their integration
```

Comment:
This seems way to strict and unwanted in my opinion. You only need to add a flex recipe if you need logic to be executed when installing your bundle. Otherwise, `type: symfony-bundle` is enough to have auto registration.

We shouldn't want to have many empty manifests in the contrib repository.

Reply:
I agree. How about:

```rst
Bundles should set ``"type": "symfony-bundle"`` in their ``composer.json`` file. With this,
`Symfony Flex`_ will be able to automatically enable your bundle when it's installed.

If your bundle requires any setup (e.g. configuration, new files, changes to `.gitignore`, etc),
then you should create a `Flex recipe`_.
```